f4se_loader/main.cpp

```diff
@@ -5,11 +5,16 @@
 #include "f4se_loader_common/IdentifyEXE.h"
 #include "f4se_loader_common/Steam.h"
 #include "f4se_loader_common/Inject.h"
+#include <algorithm>
 #include <string>
+#include <vector>
 #include "common/IFileStream.h"
+#include <Shlwapi.h>
 #include <tlhelp32.h>
 #include "Options.h"
 
+#pragma comment( lib, "Shlwapi" )

```

Comment:
```suggestion
```
Libraries should always come in from project/build config, not from pragmas.

Reply:
This is a compatibility thing with the provided vs project. The cmake project generates these dependencies automatically

Reply:
The proper way to support the vs project is to update f4se_loader.vcxproj. If you don't want to open it in devenv you can just add the dependency to both AdditionalDependencies lines along version.lib. Adding libraries via pragma linker comments adds a high element of surprise.